media/js/firefox/ios.js

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+;(function($, Mozilla) {
+    'use strict';
+
+    var $html = $('html');
+    var $body = $('body');
+
+    // send-to-device form
+    var $widget = $('#send-to-modal-container');
+    var sendToDeviceForm = new Mozilla.SendToDevice();
+
+    // Countries in which Firefox for iOS is available.
+    // Add two-letter country codes (lowercase) to this array to
+    // to expand the market. We'll remove this logic for wide release.
+    // Note: This is the country code, NOT locale code; they often differ.
+    var marketCountries = ['nz'];
```

Comment:
Nope, this is as it should be. It's a bit hacky but this array seemed like the simplest way of listing countries for the soft launch. We'll just add more countries by hand as the release goes wider. Once we go global we'll remove all of the geo stuff entirely.